inc/Integrations/Airtable/AirtableDatasource.php

```diff
@@ -96,4 +97,69 @@ public function to_ui_display(): array {
 			'uuid'    => $this->config['uuid'] ?? null,
 		];
 	}
+
+	public function ___temp_get_query(): HttpQueryContext|\WP_Error {
+		$input_schema = [
+			'record_id' => [
+				'name' => 'Record ID',
+				'type' => 'id',
+			],
+		];
+
+		$output_schema = [
+			'is_collection' => false,
+			'mappings'      => [
+				'id' => [
+					'name' => 'Record ID',
+					'path' => '$.id',
+					'type' => 'id',
+				],
+			],
+		];
+
+		foreach ( $this->config['tables'][0]['output_query_mappings'] as $mapping ) {
+			$output_schema['mappings'][ strtolower( $mapping['name'] ) ] = [
+				'name' => $mapping['name'],
+				'path' => '$.fields.' . ucfirst( $mapping['name'] ),
+				'type' => $mapping['type'] ?? 'string',
+			];
+		}
+
+		return HttpQueryContext::from_array([
+			'datasource'    => $this,
+			'input_schema'  => $input_schema,
+			'output_schema' => $output_schema,
+			'endpoint'      => $this->get_endpoint() . "/{$this->config['tables'][0]['id']}/:record_id",
+		]);
+	}
+
+	public function ___temp_get_list_query(): HttpQueryContext|\WP_Error {
+		$output_schema = [
+			'root_path'     => '$.records[*]',
+			'is_collection' => true,
+			'mappings'      => [
+				'record_id' => [
+					'name' => 'Record ID',
+					'path' => '$.id',
+					'type' => 'id',
+				],
+			],
+		];
+
+		foreach ( $this->config['tables'][0]['output_query_mappings'] as $mapping ) {
+			$output_schema['mappings'][ strtolower( $mapping['name'] ) ] = [
+				'name' => $mapping['name'],
+				'path' => '$.fields.' . ucfirst( $mapping['name'] ),
+				'type' => $mapping['type'] ?? 'string',
+			];
+		}
+
+		return HttpQueryContext::from_array([
+			'datasource'    => $this,
+			'input_schema'  => [],
+			'output_schema' => $output_schema,
+			'query_name'    => $this->config['tables'][0]['name'],
+			'endpoint'      => $this->get_endpoint() . '/' . $this->config['tables'][0]['id'],
```

Comment:
I get it. It would be nice if we could represent 90+% of queries in config. But I don't think we know enough about usage yet. We should just make an `AirtableQuery` class

Reply:
I see that differently. This is working code, the deadline is here, and it's reasonably easy to `rm -rf` it.